airflow/cli/commands/connection_command.py

```diff
@@ -325,3 +325,23 @@ def _import_helper(file_path: str, overwrite: bool) -> None:
             session.merge(conn)
             session.commit()
             print(f"Imported connection {conn_id}")
+
+
+@suppress_logs_and_warning
+def connections_test(args) -> None:
+    """Test an Airflow connection."""
+    console = AirflowConsole()
+
+    try:
+        print(f"Retrieving connection: {args.conn_id!r}")
```

Comment:
Any reason this is not outside the try block?

Reply:
Excellent point here; you're right. It should really be outside the try block.